Mathlib/Topology/Instances/ENNReal/Lemmas.lean

```diff
@@ -50,6 +50,10 @@ theorem coe_range_mem_nhds : range ((↑) : ℝ≥0 → ℝ≥0∞) ∈ 𝓝 (r
 theorem continuous_coe : Continuous ((↑) : ℝ≥0 → ℝ≥0∞) :=
   isEmbedding_coe.continuous
 
+lemma tendsto_coe_toNNReal {a : ℝ≥0∞} (ha : a ≠ ⊤) : Tendsto (↑) (𝓝 a.toNNReal) (𝓝 a) := by
+  nth_rewrite 2 [← coe_toNNReal ha]
+  exact continuous_coe.tendsto _
```

Comment:
A nice style for this kind of proofs where things match up up to some simplifications is
```suggestion
  simpa [coe_toNNReal ha] using continuous_coe.tendsto _
```
or in this case possibly even
```suggestion
  simpa [ha] using continuous_coe.tendsto _
```